doc/sr_python_walkthrough.rst

```diff
@@ -3,25 +3,23 @@
 Python
 ******
 
-This section will detail the SmartRedis Python client and how to
-use it within SmartSim applications and RedisAI in general.
+This section details the SmartRedis Python client to demonstrate its general use within SmartSim applications and RedisAI.
 
 
 .. note::
-      The Python API examples are written to connect to a
-      database at ``127.0.0.1:6379``.  When running this example,
-      ensure that the address and port of your Redis instance are used.
+      The following Python API examples connect to a
+      database at the address:port ``127.0.0.1:6379``.  When replicating the following example,
+      ensure that you use the address:port of your local Redis instance.
 
 
 
 Tensors
 =======
 
-The Python client has the ability to send and receive tensors from
-the Redis database.  The tensors are stored in the Redis database
-as RedisAI data structures.  Additionally, Python client API
+The Python client can exchange tensors from
+the Redis database stored as RedisAI data structures.  Additionally, Python client API
```

Comment:
How about: "The Python client can send and receive tensors from the Redis database, where they are stored as RedisAI data structures."

Reply:
I like that much better